### DISCUSSION TITLE
Calendar Creation steps different from Wiki

### DISCUSSION THREAD

**Discussion by zegron** [kewisch/gdata-provider#326] [Support] [answered]
The wiki says "To subscribe to one or more of your calendars and task lists, all you need to do is open the new calendar wizard (File → New → Calendar → On The Network → Google Calendar). In the next screen, you must enter the email address you use to access Google Calendar. An authentication dialog will pop up where you should log in with your Google Credentials. Two Factor Authentication is supported." 

**But thats not what is in Thunderbird when i go to access my google Calendar.**

What i see is (File → New → Calendar → On The Network → Username/Location Prompt) i put my email, as seen in the screenshot) and the location prompt is filled with my email's domain name. Which of course has no calendars to find. 

![image](https://user-images.githubusercontent.com/95231386/143885282-8742bc2f-9779-4bb4-9d45-55b3edd70b9c.png)

What do I do? How do i add my google calendar to my thunderbird?

Thanks,
Zegron

**Comment by kewisch** [comment, marked as answer]
Thanks for flagging. I've updated the wiki. You will need to go back to the initial page and select "Google Calendar" instead of "On the Network". Make sure you are using the latest version of Thunderbird and the Provider if you don't see that.
